build.zig

```diff
@@ -152,6 +152,18 @@ pub fn build(b: *std.Build) !void {
         }
     };
 
+    config.sentry = b.option(
+        bool,
+        "sentry",
+        "Build with Sentry crash reporting. Default for Linux and macOS is true, false for any other system.",
+    ) orelse sentry: {
+        if (target.result.isDarwin()) break :sentry true;
+        switch (target.result.os.tag) {
+            .linux, .macos, .ios => break :sentry true,
```

Comment:
We should probably just disable Sentry on Linux. It actually doesn't capture symbols today so its not useful.

Reply:
Done. I left the CI task that builds Sentry support on Linux to prevent any regressions.